nexus/tests/integration_tests/volume_management.rs

```diff
@@ -3566,7 +3567,7 @@ struct TestReadOnlyRegionReferenceUsage {
     datastore: Arc<DataStore>,
 
     region: db::model::Region,
-    region_address: SocketAddrV6,
+    region_address: SocketAddr,
```

Comment:
a great sadness washes over me

ok more seriously: this is because `CrucibleOpts::target`  is _also_ in terms of `SocketAddr` yeah? i'm not sweating test code nearly as much, just keeping a mental tally of what's pushing/pulling `CrucibleOpts::target`'s type.